joomla/joomla-cms
# [3.9] Actions log. Fix untranslated lang strings. Possibility to force language files loading if not loaded automatically

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/22267 , https://github.com/joomla/joomla-cms/issues/22951

### Summary of Changes
If language files are not loaded automatically:
- Establishes a new and optional property `load_language` for message object of logged actions. E.g. `'plg_user_terms'`, `'com_thatthis'`,` 'lib_something'`
- Method ActionlogsHelper::getHumanReadableLogMessage will load the corresponding language file then e.g. in control panel for module 'Action Logs - Latest' 

### Testing Instructions
- Activate plugin "User - Terms and Conditions"
- Register a new user in FE
- Go into control panel (BE)
- the latest actions module displays the language string `PLG_USER_TERMS_LOGGING_CONSENT_TO_TERMS` instead of the value.
- Apply patch.
- Register a new user in FE
- Go into control panel (BE)
- See module "Latest Actions" and check if at least the new entry is translated correctly ("user x consented to the terms and conditions during registration"). 

### Documentation Changes Required
Yes.
